ee/get-support.md

```diff
@@ -45,6 +45,26 @@ This support dump only contains logs for the node where you're running the
 command. If your UCP is highly available, you should collect support dumps
 from all of the manager nodes.
 
+## Use the CLI to get a support dump (multiple nodes)
+
+To get the support dump from the CLI for multiple nodes, use SSH to log into a UCP manager node
+and run:
+
+To generate support dump details for all hosts in a cluster, from one remote host, please see below.
+
+The below variables need to be set initially and folders created prior to running the command. Please note the variables can be changed to fit your environment:
+
+```$ SUPPORT_DUMP_DIR=<^>/opt<^^>
```

Comment:
1. Remove `$` from the beginning of the snippets. It makes it easier for users to copy-paste the snippet.
2. Move the first variable one line below
3. Make the first variable be `SUPPORT_DUMP_DIR=/tmp`

Users will understand that they can customize it, so we don't need to use that weird syntax.

Reply:
completed.